hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/streams/assertions/VisibleItemsValidator.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.bdd.spec.utilops.streams.assertions;
+
+import com.hedera.services.bdd.spec.HapiSpec;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Map;
+
+/**
+ * A validator for visible items, called once all items have been collected.
```

Comment:
What are 'visible' items in this context?

Reply:
@tinker-michaelj ⬆️

Reply:
Just items that are visible to the validator in recordstream